setup.py

```diff
@@ -103,10 +100,12 @@ def is_platform_mac():
             include_dirs=['pyuvdata/uvdata/src']
         )
     ],
-    'scripts': glob.glob('scripts/*'),
-    'version': version.version,
+    'scripts': [fl for fl in glob.glob('scripts/*') if not os.path.isdir(fl)],
+    'use_scm_version': {
+        'local_scheme': branch_scheme
+    },
     'include_package_data': True,
-    'setup_requires': ['pytest-runner', 'numpy>=1.15'],
+    # 'setup_requires': ['pytest-runner', 'numpy>=1.15'],
```

Comment:
since this is moving to the `pyproject.toml` why not delete this line entirely?

Reply:
Yup.